src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ReplyToMessage.vue

```diff
@@ -128,19 +130,29 @@ export default defineComponent({
     handleFileUpdate(filesArray: any[]) {
       this.areAttachedFilesValid = true;
       this.isFileUploadInProgress = false;
+      this.attachments = []; // Reset attachments
       if (filesArray && filesArray.length > 0) {
         for (let i = 0; i < filesArray.length; i++) {
           const file = filesArray[i];
-          if (!(file.fileErrors.length > 0) && file.progress < 101) {
+
+          // Check if file is still uploading
+          if (file.progress < 101) {
```

Comment:
Need to review file validations checks here, modifying this would cause unexpected error messages in file uploader component

Reply:
Fixed.